test_linearity_of_speed_constraints.py

```diff
@@ -0,0 +1,42 @@
+def test_linearity_of_motion_constraints():
```

Comment:
Remove this file from the PR.

Reply:
> Remove this file from the PR.

I cannot see the blue line in my code in VSC.

Reply:
You've committed a new file in the root of sympy and it shows up here. It needs to be removed.